golibs/stdgo/_internal/syscall/js/Js_value_static_extension.hx:80: funclit

/home/runner/work/go2hx/go2hx/golibs/std_js.js:10830
	throw haxe_Exception.thrown("not implemented: call");
	^
<ref *1> Error: not implemented: call
    at new haxe_ValueException (/home/runner/work/go2hx/go2hx/golibs/std_js.js:16560:17)
    at haxe_Exception.thrown (/home/runner/work/go2hx/go2hx/golibs/std_js.js:16490:11)
    at stdgo__$internal_syscall_js_Value_$static_$extension.call (/home/runner/work/go2hx/go2hx/golibs/std_js.js:10830:23)
    at /home/runner/work/go2hx/go2hx/golibs/std_js.js:46838:115
    at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_js.js:48524:3)
    at Module._compile (node:internal/modules/cjs/loader:1734:14)
    at Object..js (node:internal/modules/cjs/loader:1899:10)
    at Module.load (node:internal/modules/cjs/loader:1469:32)
    at Function._load (node:internal/modules/cjs/loader:1286:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14) {
  __previousException: undefined,
  __nativeException: [Circular *1],
  __skipStack: 2,
  value: 'not implemented: call'
}

Node.js v23.11.1
